sdk/src/types/block/mana/parameters.rs

```diff
@@ -108,8 +108,8 @@ impl ProtocolParameters {
     pub fn mana_with_decay(
         &self,
         mana: u64,
-        slot_index_created: impl Into<SlotIndex>,
-        slot_index_target: impl Into<SlotIndex>,
+        slot_index_created: impl Into<SlotIndex> + Copy,
```

Comment:
Do we actually need this bound?

Reply:
In some places yes, but I added it in too many

Reply:
I mean, even in those places we can just call `.into()` first and it's guaranteed Copy.

Reply:
You prefer having this
```Rust
    let slot_index = slot_index.into();
    let min_committable_age = min_committable_age.into();
    let max_committable_age = max_committable_age.into();
```
over the Copy bound?